### DISCUSSION TITLE
a problem occur  when i try to download the original data

### DISCUSSION THREAD

**Discussion by snow-bean** [junyanz/pytorch-CycleGAN-and-pix2pix#1502] [Q&A]
i  try  to run  this command  [ bash ./datasets/download_cyclegan_dataset.sh maps] to  download the original  data by git bash ,but i failed.

and they only give me these message below  and with  no data was download  
 Available datasets are: apple2orange, summer2winter_yosemite, horse2zebra, monet2photo, cezanne2photo, ukiyoe2photo, vangogh2photo, maps, cityscapes, facades, iphone2dslr_flower, ae_photos

where did i wrong?what should i do ? thanks for your reply

**Comment by junyanz** [comment]
Not sure what happened. You can download datasets directly from this directory. 
http://efrosgans.eecs.berkeley.edu/cyclegan/datasets/
